modules/core/shared/src/test/scala/gem/math/HourAngleSpec.scala

```diff
@@ -17,6 +17,7 @@ final class HourAngleSpec extends CatsSuite {
   // Laws
   checkAll("HourAngle", CommutativeGroupTests[HourAngle].commutativeGroup)
   checkAll("HourAngle", EqTests[HourAngle].eqv)
+  checkAll("HourAngle <: Angle", SubgroupTests[HourAngle, Angle].subgroup)
```

Comment:
I needed to read your description about what exactly an `HourAngle` was to see why it wasn't isomorphic to `Angle`, but now I get it. Yes, it should definitely embed via the identity homomorphism as a subgroup of `Angle`.

Reply:
👍